docs/data/horizon/horizon-providers.mdx

```diff
@@ -17,14 +17,14 @@ These providers allow access to the Testnet and Mainnet networks.
 
 | Provider | Access | Futurenet | Testnet | Mainnet |
 | --- | --- | --- | --- | --- |
-| [Blockdaemon\*](https://docs.blockdaemon.com/reference/supported-stellar-methods) | [Access](https://www.blockdaemon.com/apply/soroban) | Not available | Available | Available |
-| [QuickNode\*\*](https://www.quicknode.com/docs/stellar) | [Access](https://www.quicknode.com/chains/stellar) | Not available | Available | Available |
+| [Blockdaemon\*†](https://docs.blockdaemon.com/reference/supported-stellar-methods) | [Access](https://www.blockdaemon.com/apply/soroban) | Not available | Available | Available |
+| [QuickNode\*](https://www.quicknode.com/docs/stellar) | [Access](https://www.quicknode.com/chains/stellar) | Not available | Available | Available |
 | [Ankr](https://www.ankr.com/rpc/advanced-api/) | [Access](https://www.ankr.com/rpc/advanced-api/) | Not available | Available | Available |
-| [Validation Cloud](https://www.validationcloud.io/) | [Access](https://www.validationcloud.io/) | Not available | Coming Soon | Coming Soon |
+| [Validation Cloud\*](https://www.validationcloud.io/) | [Access](https://app.validationcloud.io/) | Not available | Available | Available |
 
-\*_Please note that Blockdaemon provides full historical data on its Horizon. Please note that Blockdaemon combines Horizon and RPC in the same endpoint._
+\*_Please note that Blockdaemon, Validation Cloud, and Quicknode combine Horizon and RPC in the same endpoint._
 
-\*\*_Please note that Quicknode combine Horizon and RPC in the same endpoint._
+†_Please note that Blockdaemon provides full historical data on its Horizon. Please note that Blockdaemon combines Horizon and RPC in the same endpoint._
```

Comment:
I don't think we need to duplicate the bit about Blockdaemon combining things in the note after the `†` asterisk-thingy. Let's just leave that one at:
```suggestion
†_Please note that Blockdaemon provides full historical data on its Horizon._
```

Reply:
Oh yes good point

Reply:
Updated